hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileGetContentsHandler.java

```diff
@@ -71,6 +78,19 @@ public void validate(@NonNull final QueryContext context) throws PreCheckExcepti
         }
     }
 
+    public Fees computeFees(@NonNull QueryContext queryContext) {
+        final var query = queryContext.query();
+        final var fileStore = queryContext.createStore(ReadableFileStore.class);
+        final var op = query.fileGetContentsOrThrow();
+        final var fileId = op.fileIDOrThrow();
+        final var responseType = op.headerOrElse(QueryHeader.DEFAULT).responseType();
+        final FileContents fileContents = contentFile(fileId, fileStore);
```

Comment:
We need to consider our design. If we have to do the work twice (reading the file content) just to calculate the fees, something is wrong. For now, it is ok, but we need to revisit this later.